app/views/homepage/_list_item_with_distance.haml

```diff
@@ -0,0 +1,70 @@
+- frontpage_fragment_cache("list_item", listing) do
```

Comment:
The cache key can not be "list_item" because the normal list item uses it. It has to be something unique, otherwise we will have weird cache clash bugs.